pkg/mount/mtd/types.go

```diff
@@ -24,6 +24,8 @@ type Flasher interface {
 	SyncWrite() error
 	// Close implements io.Close for a flash device.
 	Close() error
+	// DevName returns the name of the flash device
+	DevName() string
```

Comment:
I would suggest Name(), in line with how it's done in other packages.

I wonder if QueueWrite should be called QueueWriteAt since it implements io.WriterAt

Reply:
done